azure/services/inboundnatrules/inboundnatrules_test.go

```diff
@@ -179,7 +179,7 @@ func TestReconcileInboundNATRule(t *testing.T) {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
 			g := NewWithT(t)
-			t.Parallel()
+			// TODO: investigate why t.Parallel() trips the race detector here.
```

Comment:
could you please open an issue for this one?

Reply:
could potentially be a good-first-issue too

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2919